ru/turing/courses/lesson6/privalov/ddl.sql

```diff
@@ -0,0 +1,170 @@
+
+
+-- object: public.debt | type: TABLE --
+-- DROP TABLE IF EXISTS public.debt CASCADE;
+CREATE TABLE public.debt (
+	debt_id numeric(10) NOT NULL,
+	debt_name varchar(100),
+	amount numeric(10,2),
+	currency_id_currency numeric(10),
```

Comment:
```suggestion
	currency_id numeric(10),
```
Такого будет достаточно, не надо писать имя таблицы на которую ссылается фк дополнительно